build.sbt

```diff
@@ -29,11 +30,9 @@ val pomInfo = (
   </developers>
 )
 
-def excludeServlet(deps: Seq[ModuleID]) = deps.map(_.exclude("javax.servlet", "servlet-api"))
```

Comment:
This is not a good thing to do. We should not be making this choice for our users, it's nothing to do with us unless we ourselves are using servlet ourselves directly.